bodhi/tests/server/test_models.py

```diff
@@ -3248,7 +3246,7 @@ def test_comment_empty(self):
         self.assertEqual(str(exc.exception), 'You must provide either some text or feedback')
 
     def test_get_url(self):
-        self.assertEqual(self.obj.get_url(), u'updates/TurboGears-1.0.8-3.fc11')
+        self.assertEqual(self.obj.get_url(), f'updates/{self.obj.alias}')
         idx = 'a3bbe1a8f2'
         with mock.patch(target='uuid.uuid4', return_value='wat'):
             self.obj.assign_alias()
```

Comment:
Method `assign_alias` was removed.

Reply:
Yeah, I believe I've successfully removed this everywhere.